src/electionguard_cli/steps/shared/decrypt_step.py

```diff
@@ -3,19 +3,16 @@
 from electionguard_cli.cli_models import BuildElectionResults, E2eDecryptResults
```

Comment:
Use relative import

Reply:
That goes up three levels and produces a mypy linting error, see prior comment